### ISSUE TITLE
Cannot read property 'RUB' of undefined

### ISSUE THREAD

**Issue by feed4rz** [hanut/xchanger#1]
Error:
```
Error: TypeError: Cannot read property 'RUB' of undefined
    at XCHANGE.convert (.../node_modules/xchanger/index.js:141:11)
```

Code:
```js
const XCHANGER = require('xchanger')
const xchange = new XCHANGER('usd')

global.price = {
  rate: 65
}

async function getRate() {
  try {
    global.price.rate = await xchange.convert(1, 'rub')
  } catch(err) {
    console.error(err)
  }
}
getRate()
```

Versions:
```
"xchanger": "^0.1.3-a"
node v10.8.0
npm 6.2.0
```

**Comment by feed4rz** [comment]
This happens because **.sync()** returns nothing. 

Exchange rate api base url was changed to **https://api.exchangeratesapi.io/latest**

**Comment by hanut** [comment]
Closed in #2

**Comment by feed4rz** [comment]
Thanks, if possible please upload new version on npm
